libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -935,20 +936,20 @@ + (NSString *)advancedAuthStateDesc:(SFOAuthAdvancedAuthState)authState
 
 - (NSURLSession*)session {
     if (_session == nil) {
-        _session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration ephemeralSessionConfiguration]];
+        SFNetwork *network = [[SFNetwork alloc] init];
+        _session = network.activeSession;
```

Comment:
Ah, our responses crossed in the night.  That makes sense.  We may want to follow up in a future PR to integrate more with `SFNetwork` in these spots, which in `SFIdentityCoordinator` for instance could benefit from the built-in auth refresh, etc.  But this is a good start.  Thanks for the explanation.